news/38.update.rst

```diff
@@ -0,0 +1 @@
+Add a missing hidden import for ``passlib``.
```

Comment:
This is PR #39 so should be called `39.update.rst`. (Not entirely sure what difference it makes.)

Reply:
The difference is the rendering in the changelog. It links to the number in the filename.